cirq/linalg/combinators.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 
 
-def kron(*matrices: np.ndarray) -> np.ndarray:
+def kron(*matrices: Union[np.ndarray, complex, float]) -> np.ndarray:
```

Comment:
update args doc since i wouldn't guess a complex or float would be a "matrix"

Reply:
Done. Renamed to `factors` and updated the arg docs to mention scalars.